docs/contributing/accessibility/accessibility.mdx

```diff
@@ -0,0 +1,77 @@
+rather than having to always tab through the full page’s content.
+
+| WCAG Success Criteria                                                                              | Description                                                                                                                                                                                                          |
+| :------------------------------------------------------------------------------------------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [3.1.1 Language of page](https://www.w3.org/WAI/WCAG21/Understanding/language-of-page)             | Include a `lang` attribute on the `html` element of each page.                                                                                                                                                       |
+| [2.4.2 Page titled](https://www.w3.org/WAI/WCAG21/Understanding/page-titled)                       | Use a `title` element and a single `h1` to display the title on each page.                                                                                                                                           |
+| [1.3.1 Info and relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships) | Wrap content in the elements that have semantic meaning that matches the content’s purpose. Note: `div` and `span` have no semantic meaning so its best to avoid using them to wrap text. Instead use a `p` element. |
+| [1.3.1 Info and relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships) | Use logical sequence for heading levels. A `h4` should not come before the first h3 element. In addition don’t skip heading levels, use the next descending header to create structure.                              |
+| [1.3.1 Info and relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships) | Use landmark regions to communicate what content can be expected; be sure to include a nav element around the page’s navigation and `main` around the content of the page.                                           |
+| [1.3.1 Info and relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships) | Use `a` elements with `hrefs` for link; use a link when the user is navigating to a new page.                                                                                                                        |
+| [1.3.1 Info and relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships) | Use `button` elements to submit data or perform on-screen action that does not move keyboard focus.                                                                                                                  |
```

Comment:
> Use `button` elements to submit data or perform on-screen action that does not move keyboard focus. 

Use button elements to submit data or perform on-screen action**s** that **do** not move keyboard focus.